hermes-console/static/partials/modal/editSubscription.html

```diff
@@ -159,17 +159,21 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit:</small> {{su
                     </div>
                 </div>
             </div>
+
             <div class="form-group">
-                <label for="trackingEnabled" class="col-md-3 control-label">Tracking enabled</label>
+                <label for="trackingMode" class="col-md-3 control-label">Tracking mode</label>
                 <div class="col-md-9">
-                    <div class="input-group checkbox">
-                        <label>
-                            <input type="checkbox" id="trackingEnabled" name="trackingEnabled" ng-model="subscription.trackingEnabled"/>
-                        </label>
+                    <div class="input-group">
+                        <select class="form-control" id="trackingMode" name="trackingMode" ng-model="subscription.trackingMode">
+                            <option value="TRACKING_OFF">TRACKING_OFF</option>
```

Comment:
how about human friendly descriptions of these options? :)
"No tracking", "Track everything", "Track message discarding only"